sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -5110,6 +5110,15 @@ pi_result cuda_piextUSMEnqueueMemAdvise(pi_queue queue, const void *ptr,
     // CU_DEVICE_ATTRIBUTE_PAGEABLE_MEMORY_ACCESS property.
   }
 
+  unsigned int is_managed;
+  PI_CHECK_ERROR(cuPointerGetAttribute(
+      &is_managed, CU_POINTER_ATTRIBUTE_IS_MANAGED, (CUdeviceptr)ptr));
+  if (!is_managed) {
+    setErrorMessage("Prefetch hint ignored as prefetch only works with USM",
```

Comment:
```suggestion
    setErrorMessage("Memory advice ignored as memory advices only works with USM",
```

Reply:
Sorry I should have caught this. Thank you for the suggestion.